docs/tutorial/axes_title_tooltip.md

```diff
@@ -119,11 +119,22 @@ chart.animate({
 })
 ```
 
+Subtitle and footer textual element options are available for your charts.
+
+<div id="tutorial_05"></div>
+
+```javascript
+chart.animate({
+    subtitle: 'with fancy animations',
+    footer: 'some copyright info'
```

Comment:
A more frequent use case is to put source info into the footer. Maybe something like this would be a better example footer string: "Source: Vizzu tutorial"

Reply:
done: https://github.com/vizzuhq/vizzu-lib/pull/349/commits/f0e635510a92ac14d2a33eb9d1842394f3bc2d3b